interactors/repeatablefield.js

```diff
@@ -0,0 +1,47 @@
+import { Button } from '@interactors/html';
+import HTML from './baseHTML';
+
+export const RepeatableFieldAddButtonInteractor = Button.extend('repeatable field add button')
+  .selector('[data-test-repeatable-field-add-item-button]');
+
+export const RepeatableFieldDeleteButtonInteractor = Button.extend('repeatable field remove button')
+  .selector('[data-test-repeatable-field-remove-item-button]');
+
+const FieldListInteractor = HTML.extend('repeatable field list')
+  .selector('[data-test-repeatable-field-list]')
+  .locator((el, index) => (
+    [...el.querySelectorAll('li')].map((li, i) => !!i === index)
+  ));
+
+// find by legend
+export const RepeatableFieldInteractor = HTML.extend('repeatable field')
+  .selector('fieldset[data-test-repeatable-field')
+  .locator(el => el.querySelector('legend').textContent)
+  .filters({
+    id: (el) => el.id,
+    emptyMessage: (el) => el.querySelector('[class^=emptyMessage]').textContent,
+    deleteButton: (el) => !!el.querySelector('[data-test-repeatable-field-remove-item-button]'),
+    addButton: (el) => !!el.querySelector('[data-test-repeatable-field-add-item-button]'),
+    addButtonId: (el) => el.querySelector('[data-test-repeatable-field-add-item-button]').id,
```

Comment:
Are these necessary to expose? Usually I try to keep the HTML API of an interactor opaque. That means the underlying implementation of the component can change, but the tests don't have too.